deny.toml

```diff
@@ -0,0 +1,144 @@
+# See all available configuration options at:
+# https://embarkstudios.github.io/cargo-deny/index.html
+
+# --------------------------------------------------
+# General
+# --------------------------------------------------
+
+# If true, metadata will be collected with `--all-features`
+all-features = true
+# If true, metadata will be collected with `--no-default-features`
+no-default-features = true
+
+# --------------------------------------------------
+# BANS
+# --------------------------------------------------
+
+[bans]
+# The graph highlighting used when creating dotgraphs for crates.
+highlight = "all"
+# The lint level for when a crate version requirement is set to `*`.
+wildcards = "allow"
```

Comment:
Is this for when we specify dependencies in our `Cargo.toml` files using `*`? If so, I'm curious why we are allowing it. Even if this is for third-party dependencies, I'm not so sure we should allow it because sooner or later, that dependency might break.

Reply:
Do you mean in the binary and example crates? We currently just specify a path for those and we do not specify a version. I can see how that can be interpreted by `cargo deny` as `*`. We can add `version = "1"` for those and change this to `deny`.